.github/workflows/integration-tests.yml

```diff
@@ -305,27 +328,45 @@ jobs:
       options: --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video --user root
 
     steps:
-      - name: Checkout PR
-        uses: actions/checkout@v4
-        if: ${{ github.event_name == 'issue_comment' }}
-        with:
-          ref: refs/pull/${{ github.event.issue.number }}/head
-          submodules: 'true'
-
       - name: Checkout
         uses: actions/checkout@v4
-        if: ${{ github.event_name != 'issue_comment' }}
         with:
           submodules: 'true'
 
-      - name: Set ROCM ENV
+      - name: Compute build dependency cache keys
```

Comment:
I don't suppose there's a way to avoid this code duplication?

Reply:
For now probably not. It ties to the overall structure between the AMD and NVIDIA integration tests. I'd think we need to figure out how to unify them in general if that's the path we want to be on. But in general, they are based on different approaches (base machine vs docker) so even seems duplication, it should be fine. cause we may need to take different steps down the road.